[2022-07-07 14:52:55,739] {processor.py:153} INFO - Started process (PID=2786) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:52:55,740] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:52:55,741] {logging_mixin.py:115} INFO - [2022-07-07 14:52:55,741] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:52:55,764] {logging_mixin.py:115} INFO - [2022-07-07 14:52:55,764] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:52:55,764] {logging_mixin.py:115} INFO - [2022-07-07 14:52:55,764] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:52:55,765] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:52:55,798] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2022-07-07 14:52:59,232] {processor.py:153} INFO - Started process (PID=3014) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:52:59,232] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:52:59,233] {logging_mixin.py:115} INFO - [2022-07-07 14:52:59,233] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:52:59,280] {logging_mixin.py:115} INFO - [2022-07-07 14:52:59,280] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:52:59,281] {logging_mixin.py:115} INFO - [2022-07-07 14:52:59,281] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:52:59,281] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:52:59,391] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.164 seconds
[2022-07-07 14:53:04,459] {processor.py:153} INFO - Started process (PID=3242) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:04,460] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:04,461] {logging_mixin.py:115} INFO - [2022-07-07 14:53:04,460] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:04,508] {logging_mixin.py:115} INFO - [2022-07-07 14:53:04,507] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:04,508] {logging_mixin.py:115} INFO - [2022-07-07 14:53:04,508] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:04,511] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:04,556] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.103 seconds
[2022-07-07 14:53:07,353] {processor.py:153} INFO - Started process (PID=3455) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:07,353] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:07,354] {logging_mixin.py:115} INFO - [2022-07-07 14:53:07,354] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:07,369] {logging_mixin.py:115} INFO - [2022-07-07 14:53:07,369] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:07,369] {logging_mixin.py:115} INFO - [2022-07-07 14:53:07,369] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:07,369] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:07,403] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-07-07 14:53:10,059] {processor.py:153} INFO - Started process (PID=3674) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:10,059] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:10,060] {logging_mixin.py:115} INFO - [2022-07-07 14:53:10,060] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:10,141] {logging_mixin.py:115} INFO - [2022-07-07 14:53:10,138] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:10,142] {logging_mixin.py:115} INFO - [2022-07-07 14:53:10,142] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:10,143] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:10,192] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.145 seconds
[2022-07-07 14:53:12,738] {processor.py:153} INFO - Started process (PID=3887) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:12,739] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:12,740] {logging_mixin.py:115} INFO - [2022-07-07 14:53:12,740] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:12,759] {logging_mixin.py:115} INFO - [2022-07-07 14:53:12,759] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:12,760] {logging_mixin.py:115} INFO - [2022-07-07 14:53:12,760] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:12,760] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:12,795] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2022-07-07 14:53:15,159] {processor.py:153} INFO - Started process (PID=4105) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:15,160] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:15,160] {logging_mixin.py:115} INFO - [2022-07-07 14:53:15,160] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:15,176] {logging_mixin.py:115} INFO - [2022-07-07 14:53:15,175] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:15,176] {logging_mixin.py:115} INFO - [2022-07-07 14:53:15,176] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:15,176] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:15,211] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2022-07-07 14:53:17,620] {processor.py:153} INFO - Started process (PID=4321) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:17,621] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:17,621] {logging_mixin.py:115} INFO - [2022-07-07 14:53:17,621] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:17,637] {logging_mixin.py:115} INFO - [2022-07-07 14:53:17,637] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:17,637] {logging_mixin.py:115} INFO - [2022-07-07 14:53:17,637] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:17,637] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:17,669] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-07-07 14:53:20,420] {processor.py:153} INFO - Started process (PID=4536) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:20,421] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:20,422] {logging_mixin.py:115} INFO - [2022-07-07 14:53:20,421] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:20,453] {logging_mixin.py:115} INFO - [2022-07-07 14:53:20,453] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:20,454] {logging_mixin.py:115} INFO - [2022-07-07 14:53:20,454] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:20,454] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:20,522] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.105 seconds
[2022-07-07 14:53:22,968] {processor.py:153} INFO - Started process (PID=4752) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:22,969] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:22,971] {logging_mixin.py:115} INFO - [2022-07-07 14:53:22,970] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:22,994] {logging_mixin.py:115} INFO - [2022-07-07 14:53:22,994] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:22,994] {logging_mixin.py:115} INFO - [2022-07-07 14:53:22,994] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:22,995] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:23,035] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2022-07-07 14:53:26,553] {processor.py:153} INFO - Started process (PID=4969) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:26,553] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:26,554] {logging_mixin.py:115} INFO - [2022-07-07 14:53:26,554] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:26,579] {logging_mixin.py:115} INFO - [2022-07-07 14:53:26,578] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:26,579] {logging_mixin.py:115} INFO - [2022-07-07 14:53:26,579] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:26,579] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:26,618] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2022-07-07 14:53:30,926] {processor.py:153} INFO - Started process (PID=5186) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:30,932] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:30,933] {logging_mixin.py:115} INFO - [2022-07-07 14:53:30,933] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:31,015] {logging_mixin.py:115} INFO - [2022-07-07 14:53:31,014] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:31,015] {logging_mixin.py:115} INFO - [2022-07-07 14:53:31,015] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:31,015] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:31,105] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.185 seconds
[2022-07-07 14:53:36,139] {processor.py:153} INFO - Started process (PID=5407) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:36,140] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:36,141] {logging_mixin.py:115} INFO - [2022-07-07 14:53:36,141] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:36,206] {logging_mixin.py:115} INFO - [2022-07-07 14:53:36,206] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:36,207] {logging_mixin.py:115} INFO - [2022-07-07 14:53:36,206] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:36,207] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:36,310] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.176 seconds
[2022-07-07 14:53:44,535] {processor.py:153} INFO - Started process (PID=5631) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:44,536] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:44,537] {logging_mixin.py:115} INFO - [2022-07-07 14:53:44,537] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:44,631] {logging_mixin.py:115} INFO - [2022-07-07 14:53:44,631] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:44,632] {logging_mixin.py:115} INFO - [2022-07-07 14:53:44,632] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:44,632] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:44,743] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.224 seconds
[2022-07-07 14:53:52,454] {processor.py:153} INFO - Started process (PID=5847) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:52,455] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:52,456] {logging_mixin.py:115} INFO - [2022-07-07 14:53:52,456] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:52,521] {logging_mixin.py:115} INFO - [2022-07-07 14:53:52,520] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:52,521] {logging_mixin.py:115} INFO - [2022-07-07 14:53:52,521] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:52,522] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:52,612] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.176 seconds
[2022-07-07 14:53:59,739] {processor.py:153} INFO - Started process (PID=6064) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:59,739] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:53:59,740] {logging_mixin.py:115} INFO - [2022-07-07 14:53:59,740] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:59,758] {logging_mixin.py:115} INFO - [2022-07-07 14:53:59,758] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:53:59,759] {logging_mixin.py:115} INFO - [2022-07-07 14:53:59,758] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:53:59,760] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:53:59,791] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-07-07 14:54:02,791] {processor.py:153} INFO - Started process (PID=6279) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:02,793] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:02,794] {logging_mixin.py:115} INFO - [2022-07-07 14:54:02,794] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:02,821] {logging_mixin.py:115} INFO - [2022-07-07 14:54:02,820] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:02,821] {logging_mixin.py:115} INFO - [2022-07-07 14:54:02,821] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:02,821] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:02,855] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2022-07-07 14:54:06,348] {processor.py:153} INFO - Started process (PID=6495) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:06,349] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:06,350] {logging_mixin.py:115} INFO - [2022-07-07 14:54:06,350] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:06,383] {logging_mixin.py:115} INFO - [2022-07-07 14:54:06,382] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:06,383] {logging_mixin.py:115} INFO - [2022-07-07 14:54:06,383] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:06,383] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:06,422] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2022-07-07 14:54:10,550] {processor.py:153} INFO - Started process (PID=6717) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:10,550] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:10,551] {logging_mixin.py:115} INFO - [2022-07-07 14:54:10,551] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:10,587] {logging_mixin.py:115} INFO - [2022-07-07 14:54:10,586] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:10,587] {logging_mixin.py:115} INFO - [2022-07-07 14:54:10,587] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:10,587] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:10,676] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.132 seconds
[2022-07-07 14:54:16,063] {processor.py:153} INFO - Started process (PID=6939) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:16,063] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:16,064] {logging_mixin.py:115} INFO - [2022-07-07 14:54:16,064] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:16,091] {logging_mixin.py:115} INFO - [2022-07-07 14:54:16,090] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:16,092] {logging_mixin.py:115} INFO - [2022-07-07 14:54:16,092] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:16,092] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:16,128] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2022-07-07 14:54:20,383] {processor.py:153} INFO - Started process (PID=7159) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:20,384] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:20,384] {logging_mixin.py:115} INFO - [2022-07-07 14:54:20,384] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:20,415] {logging_mixin.py:115} INFO - [2022-07-07 14:54:20,414] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:20,415] {logging_mixin.py:115} INFO - [2022-07-07 14:54:20,415] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:20,415] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:20,484] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.105 seconds
[2022-07-07 14:54:24,653] {processor.py:153} INFO - Started process (PID=7375) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:24,654] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:24,655] {logging_mixin.py:115} INFO - [2022-07-07 14:54:24,655] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:24,710] {logging_mixin.py:115} INFO - [2022-07-07 14:54:24,709] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:24,710] {logging_mixin.py:115} INFO - [2022-07-07 14:54:24,710] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:24,711] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:24,769] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.123 seconds
[2022-07-07 14:54:28,599] {processor.py:153} INFO - Started process (PID=7589) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:28,600] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:28,600] {logging_mixin.py:115} INFO - [2022-07-07 14:54:28,600] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:28,620] {logging_mixin.py:115} INFO - [2022-07-07 14:54:28,619] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:28,620] {logging_mixin.py:115} INFO - [2022-07-07 14:54:28,620] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:28,620] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:28,658] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2022-07-07 14:54:31,796] {processor.py:153} INFO - Started process (PID=7812) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:31,797] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:31,797] {logging_mixin.py:115} INFO - [2022-07-07 14:54:31,797] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:31,821] {logging_mixin.py:115} INFO - [2022-07-07 14:54:31,820] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:31,821] {logging_mixin.py:115} INFO - [2022-07-07 14:54:31,821] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:31,821] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:31,861] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2022-07-07 14:54:34,574] {processor.py:153} INFO - Started process (PID=8027) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:34,574] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:34,575] {logging_mixin.py:115} INFO - [2022-07-07 14:54:34,575] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:34,594] {logging_mixin.py:115} INFO - [2022-07-07 14:54:34,593] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:34,594] {logging_mixin.py:115} INFO - [2022-07-07 14:54:34,594] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:34,594] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:34,625] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2022-07-07 14:54:37,297] {processor.py:153} INFO - Started process (PID=8243) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:37,300] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:37,301] {logging_mixin.py:115} INFO - [2022-07-07 14:54:37,301] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:37,328] {logging_mixin.py:115} INFO - [2022-07-07 14:54:37,327] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:37,328] {logging_mixin.py:115} INFO - [2022-07-07 14:54:37,328] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:37,328] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:37,370] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2022-07-07 14:54:39,729] {processor.py:153} INFO - Started process (PID=8460) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:39,729] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:39,730] {logging_mixin.py:115} INFO - [2022-07-07 14:54:39,730] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:39,746] {logging_mixin.py:115} INFO - [2022-07-07 14:54:39,746] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:39,746] {logging_mixin.py:115} INFO - [2022-07-07 14:54:39,746] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:39,747] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:39,782] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-07-07 14:54:43,033] {processor.py:153} INFO - Started process (PID=8680) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:43,033] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:43,034] {logging_mixin.py:115} INFO - [2022-07-07 14:54:43,034] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:43,061] {logging_mixin.py:115} INFO - [2022-07-07 14:54:43,060] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:43,061] {logging_mixin.py:115} INFO - [2022-07-07 14:54:43,061] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:43,061] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:43,091] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2022-07-07 14:54:46,002] {processor.py:153} INFO - Started process (PID=8903) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:46,003] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:46,004] {logging_mixin.py:115} INFO - [2022-07-07 14:54:46,004] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:46,068] {logging_mixin.py:115} INFO - [2022-07-07 14:54:46,067] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:46,068] {logging_mixin.py:115} INFO - [2022-07-07 14:54:46,068] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:46,069] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:46,133] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.149 seconds
[2022-07-07 14:54:49,458] {processor.py:153} INFO - Started process (PID=9123) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:49,459] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:49,459] {logging_mixin.py:115} INFO - [2022-07-07 14:54:49,459] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:49,475] {logging_mixin.py:115} INFO - [2022-07-07 14:54:49,475] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:49,475] {logging_mixin.py:115} INFO - [2022-07-07 14:54:49,475] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:49,475] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:49,509] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-07-07 14:54:52,431] {processor.py:153} INFO - Started process (PID=9336) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:52,432] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:52,432] {logging_mixin.py:115} INFO - [2022-07-07 14:54:52,432] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:52,447] {logging_mixin.py:115} INFO - [2022-07-07 14:54:52,447] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:52,448] {logging_mixin.py:115} INFO - [2022-07-07 14:54:52,448] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:52,448] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:52,480] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2022-07-07 14:54:56,427] {processor.py:153} INFO - Started process (PID=9552) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:56,428] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:56,429] {logging_mixin.py:115} INFO - [2022-07-07 14:54:56,429] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:56,471] {logging_mixin.py:115} INFO - [2022-07-07 14:54:56,470] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:56,471] {logging_mixin.py:115} INFO - [2022-07-07 14:54:56,471] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:56,471] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:56,512] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.103 seconds
[2022-07-07 14:54:59,599] {processor.py:153} INFO - Started process (PID=9769) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:59,599] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:54:59,600] {logging_mixin.py:115} INFO - [2022-07-07 14:54:59,600] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:59,651] {logging_mixin.py:115} INFO - [2022-07-07 14:54:59,650] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:54:59,651] {logging_mixin.py:115} INFO - [2022-07-07 14:54:59,651] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:54:59,651] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:54:59,682] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.098 seconds
[2022-07-07 14:55:02,800] {processor.py:153} INFO - Started process (PID=9986) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:02,802] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:02,803] {logging_mixin.py:115} INFO - [2022-07-07 14:55:02,803] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:02,845] {logging_mixin.py:115} INFO - [2022-07-07 14:55:02,844] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:02,845] {logging_mixin.py:115} INFO - [2022-07-07 14:55:02,845] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:02,848] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:02,912] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.126 seconds
[2022-07-07 14:55:07,513] {processor.py:153} INFO - Started process (PID=10202) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:07,514] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:07,521] {logging_mixin.py:115} INFO - [2022-07-07 14:55:07,521] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:07,574] {logging_mixin.py:115} INFO - [2022-07-07 14:55:07,573] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:07,574] {logging_mixin.py:115} INFO - [2022-07-07 14:55:07,574] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:07,575] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:07,645] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.141 seconds
[2022-07-07 14:55:12,242] {processor.py:153} INFO - Started process (PID=10418) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:12,245] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:12,246] {logging_mixin.py:115} INFO - [2022-07-07 14:55:12,246] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:12,275] {logging_mixin.py:115} INFO - [2022-07-07 14:55:12,275] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:12,276] {logging_mixin.py:115} INFO - [2022-07-07 14:55:12,276] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:12,276] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:12,316] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.087 seconds
[2022-07-07 14:55:15,800] {processor.py:153} INFO - Started process (PID=10635) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:15,801] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:15,802] {logging_mixin.py:115} INFO - [2022-07-07 14:55:15,801] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:15,817] {logging_mixin.py:115} INFO - [2022-07-07 14:55:15,816] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:15,817] {logging_mixin.py:115} INFO - [2022-07-07 14:55:15,817] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:15,817] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:15,897] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.100 seconds
[2022-07-07 14:55:18,956] {processor.py:153} INFO - Started process (PID=10858) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:18,957] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:18,958] {logging_mixin.py:115} INFO - [2022-07-07 14:55:18,958] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:18,981] {logging_mixin.py:115} INFO - [2022-07-07 14:55:18,981] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:18,981] {logging_mixin.py:115} INFO - [2022-07-07 14:55:18,981] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:18,982] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:19,023] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2022-07-07 14:55:22,024] {processor.py:153} INFO - Started process (PID=11077) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:22,024] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:22,025] {logging_mixin.py:115} INFO - [2022-07-07 14:55:22,025] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:22,043] {logging_mixin.py:115} INFO - [2022-07-07 14:55:22,042] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:22,043] {logging_mixin.py:115} INFO - [2022-07-07 14:55:22,043] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:22,043] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:22,081] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2022-07-07 14:55:25,054] {processor.py:153} INFO - Started process (PID=11295) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:25,054] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:25,055] {logging_mixin.py:115} INFO - [2022-07-07 14:55:25,055] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:25,082] {logging_mixin.py:115} INFO - [2022-07-07 14:55:25,081] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:25,082] {logging_mixin.py:115} INFO - [2022-07-07 14:55:25,082] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:25,083] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:25,122] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2022-07-07 14:55:28,757] {processor.py:153} INFO - Started process (PID=11511) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:28,759] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:28,760] {logging_mixin.py:115} INFO - [2022-07-07 14:55:28,760] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:28,781] {logging_mixin.py:115} INFO - [2022-07-07 14:55:28,781] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:28,781] {logging_mixin.py:115} INFO - [2022-07-07 14:55:28,781] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:28,782] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:28,822] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2022-07-07 14:55:32,283] {processor.py:153} INFO - Started process (PID=11727) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:32,283] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:32,284] {logging_mixin.py:115} INFO - [2022-07-07 14:55:32,284] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:32,311] {logging_mixin.py:115} INFO - [2022-07-07 14:55:32,310] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:32,311] {logging_mixin.py:115} INFO - [2022-07-07 14:55:32,311] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:32,311] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:32,364] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.087 seconds
[2022-07-07 14:55:35,713] {processor.py:153} INFO - Started process (PID=11944) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:35,713] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:35,714] {logging_mixin.py:115} INFO - [2022-07-07 14:55:35,714] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:35,730] {logging_mixin.py:115} INFO - [2022-07-07 14:55:35,730] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:35,730] {logging_mixin.py:115} INFO - [2022-07-07 14:55:35,730] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:35,731] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:35,764] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2022-07-07 14:55:40,274] {processor.py:153} INFO - Started process (PID=12160) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:40,275] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:40,275] {logging_mixin.py:115} INFO - [2022-07-07 14:55:40,275] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:40,334] {logging_mixin.py:115} INFO - [2022-07-07 14:55:40,333] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:40,334] {logging_mixin.py:115} INFO - [2022-07-07 14:55:40,334] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:40,334] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:40,426] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.167 seconds
[2022-07-07 14:55:46,748] {processor.py:153} INFO - Started process (PID=12377) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:46,749] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:46,749] {logging_mixin.py:115} INFO - [2022-07-07 14:55:46,749] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:46,782] {logging_mixin.py:115} INFO - [2022-07-07 14:55:46,782] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:46,783] {logging_mixin.py:115} INFO - [2022-07-07 14:55:46,783] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:46,783] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:46,901] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.156 seconds
[2022-07-07 14:55:54,435] {processor.py:153} INFO - Started process (PID=12593) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:54,436] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:55:54,436] {logging_mixin.py:115} INFO - [2022-07-07 14:55:54,436] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:54,509] {logging_mixin.py:115} INFO - [2022-07-07 14:55:54,509] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:55:54,510] {logging_mixin.py:115} INFO - [2022-07-07 14:55:54,510] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:55:54,510] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:55:54,575] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.185 seconds
[2022-07-07 14:56:00,696] {processor.py:153} INFO - Started process (PID=12816) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:00,697] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:00,710] {logging_mixin.py:115} INFO - [2022-07-07 14:56:00,710] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:00,789] {logging_mixin.py:115} INFO - [2022-07-07 14:56:00,789] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:00,790] {logging_mixin.py:115} INFO - [2022-07-07 14:56:00,790] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:00,790] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:00,875] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.187 seconds
[2022-07-07 14:56:04,854] {processor.py:153} INFO - Started process (PID=13038) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:04,854] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:04,855] {logging_mixin.py:115} INFO - [2022-07-07 14:56:04,855] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:04,879] {logging_mixin.py:115} INFO - [2022-07-07 14:56:04,879] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:04,880] {logging_mixin.py:115} INFO - [2022-07-07 14:56:04,880] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:04,880] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:04,924] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2022-07-07 14:56:08,502] {processor.py:153} INFO - Started process (PID=13254) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:08,503] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:08,504] {logging_mixin.py:115} INFO - [2022-07-07 14:56:08,503] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:08,569] {logging_mixin.py:115} INFO - [2022-07-07 14:56:08,558] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:08,570] {logging_mixin.py:115} INFO - [2022-07-07 14:56:08,569] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:08,570] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:08,636] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.144 seconds
[2022-07-07 14:56:12,341] {processor.py:153} INFO - Started process (PID=13470) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:12,342] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:12,342] {logging_mixin.py:115} INFO - [2022-07-07 14:56:12,342] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:12,357] {logging_mixin.py:115} INFO - [2022-07-07 14:56:12,357] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:12,358] {logging_mixin.py:115} INFO - [2022-07-07 14:56:12,358] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:12,358] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:12,391] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-07-07 14:56:15,760] {processor.py:153} INFO - Started process (PID=13686) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:15,761] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:15,762] {logging_mixin.py:115} INFO - [2022-07-07 14:56:15,761] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:15,797] {logging_mixin.py:115} INFO - [2022-07-07 14:56:15,796] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:15,798] {logging_mixin.py:115} INFO - [2022-07-07 14:56:15,797] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:15,798] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:15,887] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.130 seconds
[2022-07-07 14:56:19,574] {processor.py:153} INFO - Started process (PID=13903) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:19,576] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:19,577] {logging_mixin.py:115} INFO - [2022-07-07 14:56:19,577] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:19,597] {logging_mixin.py:115} INFO - [2022-07-07 14:56:19,596] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:19,597] {logging_mixin.py:115} INFO - [2022-07-07 14:56:19,597] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:19,597] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:19,640] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.071 seconds
[2022-07-07 14:56:22,878] {processor.py:153} INFO - Started process (PID=14125) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:22,879] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:22,879] {logging_mixin.py:115} INFO - [2022-07-07 14:56:22,879] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:22,901] {logging_mixin.py:115} INFO - [2022-07-07 14:56:22,901] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:22,901] {logging_mixin.py:115} INFO - [2022-07-07 14:56:22,901] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:22,901] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:22,934] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2022-07-07 14:56:26,027] {processor.py:153} INFO - Started process (PID=14343) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:26,027] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:26,028] {logging_mixin.py:115} INFO - [2022-07-07 14:56:26,028] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:26,044] {logging_mixin.py:115} INFO - [2022-07-07 14:56:26,044] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:26,044] {logging_mixin.py:115} INFO - [2022-07-07 14:56:26,044] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:26,045] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:26,077] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2022-07-07 14:56:28,921] {processor.py:153} INFO - Started process (PID=14557) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:28,922] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:28,923] {logging_mixin.py:115} INFO - [2022-07-07 14:56:28,922] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:28,943] {logging_mixin.py:115} INFO - [2022-07-07 14:56:28,943] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:28,943] {logging_mixin.py:115} INFO - [2022-07-07 14:56:28,943] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:28,944] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:28,984] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2022-07-07 14:56:32,207] {processor.py:153} INFO - Started process (PID=14780) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:32,208] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:32,209] {logging_mixin.py:115} INFO - [2022-07-07 14:56:32,208] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:32,227] {logging_mixin.py:115} INFO - [2022-07-07 14:56:32,226] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:32,227] {logging_mixin.py:115} INFO - [2022-07-07 14:56:32,227] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:32,227] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:32,266] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2022-07-07 14:56:35,444] {processor.py:153} INFO - Started process (PID=15002) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:35,444] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:35,445] {logging_mixin.py:115} INFO - [2022-07-07 14:56:35,445] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:35,467] {logging_mixin.py:115} INFO - [2022-07-07 14:56:35,466] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:35,467] {logging_mixin.py:115} INFO - [2022-07-07 14:56:35,467] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:35,467] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:35,500] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2022-07-07 14:56:38,651] {processor.py:153} INFO - Started process (PID=15218) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:38,652] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:38,653] {logging_mixin.py:115} INFO - [2022-07-07 14:56:38,653] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:38,674] {logging_mixin.py:115} INFO - [2022-07-07 14:56:38,674] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:38,674] {logging_mixin.py:115} INFO - [2022-07-07 14:56:38,674] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:38,675] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:38,706] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2022-07-07 14:56:41,664] {processor.py:153} INFO - Started process (PID=15434) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:41,665] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:41,666] {logging_mixin.py:115} INFO - [2022-07-07 14:56:41,666] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:41,692] {logging_mixin.py:115} INFO - [2022-07-07 14:56:41,691] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:41,692] {logging_mixin.py:115} INFO - [2022-07-07 14:56:41,692] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:41,692] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:41,744] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2022-07-07 14:56:44,786] {processor.py:153} INFO - Started process (PID=15650) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:44,786] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:44,787] {logging_mixin.py:115} INFO - [2022-07-07 14:56:44,786] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:44,804] {logging_mixin.py:115} INFO - [2022-07-07 14:56:44,804] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:44,804] {logging_mixin.py:115} INFO - [2022-07-07 14:56:44,804] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:44,805] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:44,844] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2022-07-07 14:56:47,793] {processor.py:153} INFO - Started process (PID=15869) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:47,793] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:47,794] {logging_mixin.py:115} INFO - [2022-07-07 14:56:47,794] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:47,816] {logging_mixin.py:115} INFO - [2022-07-07 14:56:47,816] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:47,817] {logging_mixin.py:115} INFO - [2022-07-07 14:56:47,817] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:47,817] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:47,854] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2022-07-07 14:56:51,004] {processor.py:153} INFO - Started process (PID=16083) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:51,004] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:51,005] {logging_mixin.py:115} INFO - [2022-07-07 14:56:51,005] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:51,025] {logging_mixin.py:115} INFO - [2022-07-07 14:56:51,024] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:51,025] {logging_mixin.py:115} INFO - [2022-07-07 14:56:51,025] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:51,025] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:51,062] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2022-07-07 14:56:54,032] {processor.py:153} INFO - Started process (PID=16301) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:54,033] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:54,034] {logging_mixin.py:115} INFO - [2022-07-07 14:56:54,034] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:54,056] {logging_mixin.py:115} INFO - [2022-07-07 14:56:54,056] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:54,057] {logging_mixin.py:115} INFO - [2022-07-07 14:56:54,057] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:54,057] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:54,106] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2022-07-07 14:56:56,848] {processor.py:153} INFO - Started process (PID=16514) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:56,849] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:56:56,851] {logging_mixin.py:115} INFO - [2022-07-07 14:56:56,851] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:56,884] {logging_mixin.py:115} INFO - [2022-07-07 14:56:56,883] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:56:56,884] {logging_mixin.py:115} INFO - [2022-07-07 14:56:56,884] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:56:56,884] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:56:56,936] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.092 seconds
[2022-07-07 14:57:01,103] {processor.py:153} INFO - Started process (PID=16731) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:01,104] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:01,105] {logging_mixin.py:115} INFO - [2022-07-07 14:57:01,105] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:01,131] {logging_mixin.py:115} INFO - [2022-07-07 14:57:01,130] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:01,131] {logging_mixin.py:115} INFO - [2022-07-07 14:57:01,131] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:01,131] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:01,197] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.100 seconds
[2022-07-07 14:57:04,512] {processor.py:153} INFO - Started process (PID=16947) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:04,514] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:04,516] {logging_mixin.py:115} INFO - [2022-07-07 14:57:04,516] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:04,537] {logging_mixin.py:115} INFO - [2022-07-07 14:57:04,537] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:04,538] {logging_mixin.py:115} INFO - [2022-07-07 14:57:04,538] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:04,538] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:04,571] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2022-07-07 14:57:07,681] {processor.py:153} INFO - Started process (PID=17170) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:07,681] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:07,682] {logging_mixin.py:115} INFO - [2022-07-07 14:57:07,682] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:07,707] {logging_mixin.py:115} INFO - [2022-07-07 14:57:07,706] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:07,707] {logging_mixin.py:115} INFO - [2022-07-07 14:57:07,707] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:07,707] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:07,739] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2022-07-07 14:57:10,670] {processor.py:153} INFO - Started process (PID=17390) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:10,671] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:10,672] {logging_mixin.py:115} INFO - [2022-07-07 14:57:10,672] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:10,693] {logging_mixin.py:115} INFO - [2022-07-07 14:57:10,693] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:10,693] {logging_mixin.py:115} INFO - [2022-07-07 14:57:10,693] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:10,694] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:10,724] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2022-07-07 14:57:13,609] {processor.py:153} INFO - Started process (PID=17608) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:13,610] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:13,610] {logging_mixin.py:115} INFO - [2022-07-07 14:57:13,610] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:13,632] {logging_mixin.py:115} INFO - [2022-07-07 14:57:13,632] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:13,632] {logging_mixin.py:115} INFO - [2022-07-07 14:57:13,632] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:13,633] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:13,671] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2022-07-07 14:57:16,876] {processor.py:153} INFO - Started process (PID=17826) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:16,877] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:16,878] {logging_mixin.py:115} INFO - [2022-07-07 14:57:16,878] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:16,895] {logging_mixin.py:115} INFO - [2022-07-07 14:57:16,894] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:16,895] {logging_mixin.py:115} INFO - [2022-07-07 14:57:16,895] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:16,895] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:16,929] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2022-07-07 14:57:19,996] {processor.py:153} INFO - Started process (PID=18041) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:19,996] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:19,997] {logging_mixin.py:115} INFO - [2022-07-07 14:57:19,997] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:20,019] {logging_mixin.py:115} INFO - [2022-07-07 14:57:20,018] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:20,019] {logging_mixin.py:115} INFO - [2022-07-07 14:57:20,019] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:20,019] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:20,061] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2022-07-07 14:57:22,944] {processor.py:153} INFO - Started process (PID=18256) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:22,944] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:22,945] {logging_mixin.py:115} INFO - [2022-07-07 14:57:22,945] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:22,974] {logging_mixin.py:115} INFO - [2022-07-07 14:57:22,974] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:22,974] {logging_mixin.py:115} INFO - [2022-07-07 14:57:22,974] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:22,974] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:23,020] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.084 seconds
[2022-07-07 14:57:26,137] {processor.py:153} INFO - Started process (PID=18472) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:26,138] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:26,139] {logging_mixin.py:115} INFO - [2022-07-07 14:57:26,139] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:26,162] {logging_mixin.py:115} INFO - [2022-07-07 14:57:26,162] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:26,162] {logging_mixin.py:115} INFO - [2022-07-07 14:57:26,162] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:26,162] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:26,203] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2022-07-07 14:57:29,355] {processor.py:153} INFO - Started process (PID=18690) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:29,357] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:29,358] {logging_mixin.py:115} INFO - [2022-07-07 14:57:29,357] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:29,382] {logging_mixin.py:115} INFO - [2022-07-07 14:57:29,382] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:29,383] {logging_mixin.py:115} INFO - [2022-07-07 14:57:29,383] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:29,383] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:29,418] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2022-07-07 14:57:32,416] {processor.py:153} INFO - Started process (PID=18906) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:32,417] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:32,418] {logging_mixin.py:115} INFO - [2022-07-07 14:57:32,417] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:32,439] {logging_mixin.py:115} INFO - [2022-07-07 14:57:32,439] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:32,440] {logging_mixin.py:115} INFO - [2022-07-07 14:57:32,440] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:32,440] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:32,469] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2022-07-07 14:57:35,544] {processor.py:153} INFO - Started process (PID=19124) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:35,545] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:35,545] {logging_mixin.py:115} INFO - [2022-07-07 14:57:35,545] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:35,562] {logging_mixin.py:115} INFO - [2022-07-07 14:57:35,561] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:35,562] {logging_mixin.py:115} INFO - [2022-07-07 14:57:35,562] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:35,562] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:35,592] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2022-07-07 14:57:38,777] {processor.py:153} INFO - Started process (PID=19338) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:38,778] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:38,778] {logging_mixin.py:115} INFO - [2022-07-07 14:57:38,778] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:38,810] {logging_mixin.py:115} INFO - [2022-07-07 14:57:38,809] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:38,810] {logging_mixin.py:115} INFO - [2022-07-07 14:57:38,810] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:38,811] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:38,870] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.103 seconds
[2022-07-07 14:57:41,879] {processor.py:153} INFO - Started process (PID=19560) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:41,880] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:41,881] {logging_mixin.py:115} INFO - [2022-07-07 14:57:41,881] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:41,908] {logging_mixin.py:115} INFO - [2022-07-07 14:57:41,908] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:41,909] {logging_mixin.py:115} INFO - [2022-07-07 14:57:41,909] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:41,909] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:41,943] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2022-07-07 14:57:44,669] {processor.py:153} INFO - Started process (PID=19779) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:44,670] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:44,670] {logging_mixin.py:115} INFO - [2022-07-07 14:57:44,670] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:44,689] {logging_mixin.py:115} INFO - [2022-07-07 14:57:44,688] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:44,689] {logging_mixin.py:115} INFO - [2022-07-07 14:57:44,689] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:44,689] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:44,724] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2022-07-07 14:57:47,471] {processor.py:153} INFO - Started process (PID=19995) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:47,472] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:47,472] {logging_mixin.py:115} INFO - [2022-07-07 14:57:47,472] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:47,494] {logging_mixin.py:115} INFO - [2022-07-07 14:57:47,493] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:47,494] {logging_mixin.py:115} INFO - [2022-07-07 14:57:47,494] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:47,494] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:47,524] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2022-07-07 14:57:50,387] {processor.py:153} INFO - Started process (PID=20211) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:50,388] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:50,388] {logging_mixin.py:115} INFO - [2022-07-07 14:57:50,388] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:50,406] {logging_mixin.py:115} INFO - [2022-07-07 14:57:50,406] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:50,406] {logging_mixin.py:115} INFO - [2022-07-07 14:57:50,406] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:50,406] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:50,440] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-07-07 14:57:54,033] {processor.py:153} INFO - Started process (PID=20425) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:54,034] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:54,034] {logging_mixin.py:115} INFO - [2022-07-07 14:57:54,034] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:54,055] {logging_mixin.py:115} INFO - [2022-07-07 14:57:54,054] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:54,055] {logging_mixin.py:115} INFO - [2022-07-07 14:57:54,055] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:54,055] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:54,091] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2022-07-07 14:57:56,952] {processor.py:153} INFO - Started process (PID=20641) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:56,953] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:56,953] {logging_mixin.py:115} INFO - [2022-07-07 14:57:56,953] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:56,970] {logging_mixin.py:115} INFO - [2022-07-07 14:57:56,970] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:56,971] {logging_mixin.py:115} INFO - [2022-07-07 14:57:56,970] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:56,971] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:57,007] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2022-07-07 14:57:59,799] {processor.py:153} INFO - Started process (PID=20858) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:59,800] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:57:59,801] {logging_mixin.py:115} INFO - [2022-07-07 14:57:59,801] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:59,824] {logging_mixin.py:115} INFO - [2022-07-07 14:57:59,824] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:57:59,825] {logging_mixin.py:115} INFO - [2022-07-07 14:57:59,825] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:57:59,825] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:57:59,866] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2022-07-07 14:58:04,104] {processor.py:153} INFO - Started process (PID=21074) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:04,105] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:04,105] {logging_mixin.py:115} INFO - [2022-07-07 14:58:04,105] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:04,128] {logging_mixin.py:115} INFO - [2022-07-07 14:58:04,127] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:04,129] {logging_mixin.py:115} INFO - [2022-07-07 14:58:04,129] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:04,129] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:04,184] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2022-07-07 14:58:07,622] {processor.py:153} INFO - Started process (PID=21289) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:07,623] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:07,623] {logging_mixin.py:115} INFO - [2022-07-07 14:58:07,623] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:07,657] {logging_mixin.py:115} INFO - [2022-07-07 14:58:07,657] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:07,657] {logging_mixin.py:115} INFO - [2022-07-07 14:58:07,657] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:07,658] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:07,711] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.094 seconds
[2022-07-07 14:58:10,912] {processor.py:153} INFO - Started process (PID=21505) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:10,913] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:10,914] {logging_mixin.py:115} INFO - [2022-07-07 14:58:10,913] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:10,936] {logging_mixin.py:115} INFO - [2022-07-07 14:58:10,936] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:10,937] {logging_mixin.py:115} INFO - [2022-07-07 14:58:10,936] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:10,937] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:10,990] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.084 seconds
[2022-07-07 14:58:14,270] {processor.py:153} INFO - Started process (PID=21722) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:14,272] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:14,277] {logging_mixin.py:115} INFO - [2022-07-07 14:58:14,277] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:14,329] {logging_mixin.py:115} INFO - [2022-07-07 14:58:14,328] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:14,329] {logging_mixin.py:115} INFO - [2022-07-07 14:58:14,329] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:14,329] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:14,405] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.140 seconds
[2022-07-07 14:58:18,354] {processor.py:153} INFO - Started process (PID=21945) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:18,355] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:18,357] {logging_mixin.py:115} INFO - [2022-07-07 14:58:18,357] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:18,411] {logging_mixin.py:115} INFO - [2022-07-07 14:58:18,410] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:18,411] {logging_mixin.py:115} INFO - [2022-07-07 14:58:18,411] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:18,412] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:18,454] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.107 seconds
[2022-07-07 14:58:22,046] {processor.py:153} INFO - Started process (PID=22166) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:22,047] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:22,048] {logging_mixin.py:115} INFO - [2022-07-07 14:58:22,048] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:22,087] {logging_mixin.py:115} INFO - [2022-07-07 14:58:22,087] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:22,087] {logging_mixin.py:115} INFO - [2022-07-07 14:58:22,087] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:22,088] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:22,170] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.137 seconds
[2022-07-07 14:58:25,774] {processor.py:153} INFO - Started process (PID=22385) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:25,775] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:25,775] {logging_mixin.py:115} INFO - [2022-07-07 14:58:25,775] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:25,800] {logging_mixin.py:115} INFO - [2022-07-07 14:58:25,800] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:25,800] {logging_mixin.py:115} INFO - [2022-07-07 14:58:25,800] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:25,800] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:25,842] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2022-07-07 14:58:29,372] {processor.py:153} INFO - Started process (PID=22600) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:29,373] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:29,373] {logging_mixin.py:115} INFO - [2022-07-07 14:58:29,373] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:29,406] {logging_mixin.py:115} INFO - [2022-07-07 14:58:29,405] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:29,406] {logging_mixin.py:115} INFO - [2022-07-07 14:58:29,406] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:29,406] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:29,447] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.084 seconds
[2022-07-07 14:58:32,857] {processor.py:153} INFO - Started process (PID=22816) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:32,858] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:32,859] {logging_mixin.py:115} INFO - [2022-07-07 14:58:32,859] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:32,893] {logging_mixin.py:115} INFO - [2022-07-07 14:58:32,893] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:32,894] {logging_mixin.py:115} INFO - [2022-07-07 14:58:32,894] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:32,894] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:32,933] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2022-07-07 14:58:36,539] {processor.py:153} INFO - Started process (PID=23032) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:36,540] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:36,541] {logging_mixin.py:115} INFO - [2022-07-07 14:58:36,541] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:36,574] {logging_mixin.py:115} INFO - [2022-07-07 14:58:36,573] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:36,574] {logging_mixin.py:115} INFO - [2022-07-07 14:58:36,574] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:36,574] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:36,610] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2022-07-07 14:58:40,386] {processor.py:153} INFO - Started process (PID=23250) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:40,387] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:40,390] {logging_mixin.py:115} INFO - [2022-07-07 14:58:40,390] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:40,412] {logging_mixin.py:115} INFO - [2022-07-07 14:58:40,411] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:40,412] {logging_mixin.py:115} INFO - [2022-07-07 14:58:40,412] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:40,412] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:40,450] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2022-07-07 14:58:43,381] {processor.py:153} INFO - Started process (PID=23466) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:43,382] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:43,383] {logging_mixin.py:115} INFO - [2022-07-07 14:58:43,383] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:43,399] {logging_mixin.py:115} INFO - [2022-07-07 14:58:43,399] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:43,399] {logging_mixin.py:115} INFO - [2022-07-07 14:58:43,399] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:43,400] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:43,433] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-07-07 14:58:46,346] {processor.py:153} INFO - Started process (PID=23680) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:46,347] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:46,348] {logging_mixin.py:115} INFO - [2022-07-07 14:58:46,347] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:46,366] {logging_mixin.py:115} INFO - [2022-07-07 14:58:46,366] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:46,367] {logging_mixin.py:115} INFO - [2022-07-07 14:58:46,367] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:46,367] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:46,411] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2022-07-07 14:58:49,614] {processor.py:153} INFO - Started process (PID=23897) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:49,615] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:49,615] {logging_mixin.py:115} INFO - [2022-07-07 14:58:49,615] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:49,632] {logging_mixin.py:115} INFO - [2022-07-07 14:58:49,632] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:49,632] {logging_mixin.py:115} INFO - [2022-07-07 14:58:49,632] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:49,632] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:49,689] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2022-07-07 14:58:53,063] {processor.py:153} INFO - Started process (PID=24124) to work on /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:53,063] {processor.py:641} INFO - Processing file /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-07 14:58:53,064] {logging_mixin.py:115} INFO - [2022-07-07 14:58:53,064] {dagbag.py:507} INFO - Filling up the DagBag from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:53,089] {logging_mixin.py:115} INFO - [2022-07-07 14:58:53,089] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-07 14:58:53,090] {logging_mixin.py:115} INFO - [2022-07-07 14:58:53,090] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-07 14:58:53,090] {processor.py:653} WARNING - No viable dags retrieved from /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-07 14:58:53,132] {processor.py:161} INFO - Processing /home/luan/.local/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
